cfspopcon/file_io.py

```diff
@@ -23,7 +23,7 @@
 ]
 
 
-def sanitize_variable(val: xr.DataArray, key: str) -> xr.DataArray:
+def sanitize_variable(val: xr.DataArray, key: str, coord: bool = False) -> xr.DataArray:
     """Strip units and Enum values from a variable so that it can be stored in a NetCDF file."""
```

Comment:
Mind adding something to the docstrnig about what `coord:bool` controls?

Reply:
Done